libioc/ResourceUpdater.py

```diff
@@ -603,12 +612,17 @@ def _modify_updater_config(self, path: str) -> None:
     def _wrap_command(self, command: str, kind: str) -> str:
 
         if kind == "update":
+            symlink_command = f"ln -s / {self._base_release_symlink_location}"
             tolerated_error_message = (
                 "echo $OUTPUT"
                 " | grep -c 'No updates are available to install.'"
                 " >> /dev/null || exit $RC"
             )
         elif kind == "fetch":
+            symlink_command = (
+                f"ln -s  {self.release.root_path}/.zfs/snapshot/p0 "
+                f"{self._base_release_symlink_location}"
+            )
```

Comment:
what's happening here?

Reply:
The symlink `self._base_release_symlink_location` is created on the host to fetch updates an in the jail to install them. When the BASEDIR `f"-b {self._base_release_symlink_location}"` is not the same, updates cannot be installed.

Reply:
why is it always `p0`?

Reply:
Because we base all updates on the unpatched release, so that jails of any patchlevel can update from the fetched assets.